common/src/Microsoft.Azure.IIoT.Hub.Module.Framework/src/Hosting/ModuleHost.cs

```diff
@@ -538,10 +540,11 @@ public sealed class ModuleHost : IModuleHost, ITwinProperties, IEventEmitter,
         private readonly SemaphoreSlim _lock = new SemaphoreSlim(1, 1);
         private readonly Dictionary<string, VariantValue> _reported =
             new Dictionary<string, VariantValue>();
+        private readonly string ModuleGuid = Guid.NewGuid().ToString();
         private static readonly Gauge kModuleStart = Metrics
             .CreateGauge("iiot_edge_module_start", "starting module",
                 new GaugeConfiguration {
-                    LabelNames = new[] {"deviceid", "module", "timestamp_utc" }
+                    LabelNames = new[] {"deviceid", "module", "guid", "version", "timestamp_utc" }
```

Comment:
>"guid" [](start = 62, length = 6)

is this label that is visible to the customer? if yes, perhaps we should find a better name